src/monattack.cpp

```diff
@@ -5168,8 +5168,9 @@ bool mattack::bio_op_takedown( monster *z )
     // Weak kick to start with, knocks you off your footing
 
     // TODO: Literally "The zombie kicks" vvvvv | Fix message or comment why Literally.
```

Comment:
This comment looks outdated now.

Reply:
O, that's right. I'm removing this comment line since the message has been fixed.